app/src/main/java/com/orange/ods/app/ui/components/cards/CardHorizontal.kt

```diff
@@ -77,7 +77,7 @@ fun CardHorizontal(customizationState: CardCustomizationState) {
                     name = OdsComposable.OdsHorizontalCard.name,
                     exhaustiveParameters = false,
                     parameters = {
-                        simple("imagePosition", imagePosition.value.name)
+                        enum("imagePosition", imagePosition.value)
```

Comment:
Here the string displayed is `imagePosition = Start`.
On the text button screen, we use ` enum("style", textButtonStyle.value)` but the string displayed is `style = OdsTextButtonStyle.Default`.
Do you know why here we don't have the `OdsHorizontalCardImagePosition.`?

Reply:
I have `OdsHorizontalCardImagePosition.Start` on my device:

![Screenshot_20230920_120422](https://github.com/Orange-OpenSource/ods-android/assets/13621389/3ff74a8e-a732-489e-b299-e9d2ff472b88)

Could you please retest?

Reply:
Yes, it's ok not :)